extensions/amp-analytics/analytics-vars.md

```diff
@@ -31,7 +31,7 @@ Vars can be defined by the platform, in the config at the top level, inside the
 </amp-analytics>
 ```
```

Comment:
As noted in a comment below, please add the data elem documentation right after this block, including the header and the code sample.